src/eventhubs/azext_eventhub/_params.py

```diff
@@ -67,7 +67,7 @@ def load_arguments_eventhub(self, _):
             c.argument('event_hub_name', options_list=['--event-hub-name'], help='name of the EventHub')
 
     with self.argument_context('eventhubs eventhub authorizationrule create') as c:
-        c.argument('accessrights', options_list=['--access-rights'], help='AuthorizationRule rights of type list, allowed values are Send, Listen or Manage')
+        c.argument('accessrights', options_list=['--access-rights'], arg_type=get_enum_type(['Send', 'Listen', 'Manage']), help='AuthorizationRule rights of type list, allowed values are Send, Listen or Manage')
```

Comment:
Please now remove `, allowed values are Send, Listen or Manage')` otherwise it will appear twice.

Reply:
my bad. corrected the help text.